input/data/field/c1f2_alfalfa_2020_field.json

```
@@ -0,0 +1,15 @@
+
+{
+  "soil_specification": "soil",
+  "crop_specification": "Alfalfa 2020 Crop data",
+  "manure_management_specification": "Alfalfa 2020 Manure schedule",
+  "tillage_management_specification": "Alfalfa 2020 Tillage schedule",
+  "field_size": 540,
```

```suggestion
  "fertilizer_management_specification": "no_fert",
  "field_size": 540,
```
Even if there is not going to be any fertilizer applied, that still has to be specified. The above line can be added to each new field input file, and the following input added to the "files" section of the metadata:
```json
    "no_fert": {
      "title": "Fertilizer schedule.",
      "description": "No fertilizer is scheduled to be used.",
      "path": "input/data/fertilizer_schedule/no_fertilizer.json",
      "type": "json",
      "properties": "fertilizer_schedule_properties"
    },
```